webpack/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lektor-example",
```

Comment:
could you rename this something less generic?

Reply:
changed to: creative-commons

Reply:
could you change to creative-commons-open-source since that's the title of the site?

Reply:
[https://github.com/creativecommons/creativecommons.github.io-source/pull/31/commits/a605323a3487aa76c652ab46339d871772f43fe9](url)

Reply:
done!

Reply:
https://github.com/creativecommons/creativecommons.github.io-source/pull/31/commits/a605323a3487aa76c652ab46339d871772f43fe9